### DISCUSSION TITLE
Gridlabd build issue with helics 2.6.0

### DISCUSSION THREAD

**Discussion by asinghal91** [GMLC-TDC/HELICS#1576] [Q&A] [answered]
If we build gridlabd from source with **helics 2.6.0**, simulation never enters into execution mode or if it does, it gets stuck at h.helicsFederateRequestTime(fed, t).

If we build gridlabd with **helics 2.4.0**, everything works fine.

Note that once gridlabd built with helisc 2.4.0, it works fine with both helics 2.4.0 and helics 2.6.0 python packages.

**Comment by trevorhardy** [comment]
Probably something for @afisher1 to look into. Any thoughts, Andy?

**Comment by trevorhardy** [comment]
@asinghal91; has this issue been resolved?

**Reply by asinghal91** [reply]
Since it is working with helics2.4.0 and I am happy with it, I did not try again with helics2.6.0. So, I am not sure if the issue is resolved or not.

**Comment by trevorhardy** [comment, marked as answer]
@asinghal91, I'm going to close out this discussion. If you are still needing help with HELICS, let me know. 

By the way, [v3 has been released ](https://github.com/GMLC-TDC/HELICS/releases)and we are recommending all users that are working on projects that will be around for many more months or years make the switch. Generally, the changes are relatively minor.
